tests/ServerTest.php

```diff
@@ -26,17 +27,14 @@ public function testRequestEvent()
         $io = new ServerStub();
 
         $i = 0;
+        $requestAssertion = null;
+        $responseAssertion = null;
 
         $server = new Server($io);
-        $server->on('request', function ($request, $response) use (&$i) {
+        $server->on('request', function (Request $request, Response $response) use (&$i, &$requestAssertion, &$responseAssertion) {
```

Comment:
Type hints should probably be omitted in this test, because they're verified as part of the assertions below? (They're okay in the others tests though)